.github/workflows/laravel.yml

```diff
@@ -0,0 +1,32 @@
+name: Laravel
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  laravel-tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: shivammathur/setup-php@v2
```

Comment:
can we split it into several jobs? e.g. `setup`, `test`, `upload-to-coveralls`

ref: https://docs.github.com/en/actions/using-workflows/about-workflows#creating-dependent-jobs

Reply:
As we discussed, I've moved this one for the separate issue https://github.com/RonasIT/laravel-helpers/issues/51https://github.com/RonasIT/laravel-helpers/issues/51